src/styles/theme/oc-spinner.scss

```diff
@@ -32,32 +30,37 @@ $oc-spinner-size: 24px;
     position: relative;
   }
 
-  &,
-  &-small {
-    @include oc-spinner-size(1);
+  @include oc-spinner-size(1);
```

Comment:
Codacy found an issue: [Expected blockless @include to come before rule (order/order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)